package fi.lipp.blog.model.exceptions

class AlreadyFollowingException : BlogException("Already following this user", 400)